RobustPGO/RobustSolver.cpp

```diff
@@ -94,7 +102,7 @@ void RobustSolver::optimize() {
 }
 
 void RobustSolver::update(const gtsam::NonlinearFactorGraph& nfg,
-                       const gtsam::Values& values) {
+                          const gtsam::Values& values) {
```

Comment:
Make this the same convention as the rest of the codebase - arguments on the next line of the declaration should be indented four spaces over.

```suggestion
    const gtsam::Values& values) {
```

Reply:
@marcusabate  wait to double check, isn't the convention to align the argument with what is on the first line? Four spaces indentation is when there is no arguments on the first line right? https://google.github.io/styleguide/cppguide.html#Reference_Arguments

Reply:
Ah you are correct, let me go through again real quick.